### PR TITLE
use recommended project settings fixes #142

### LINKED ISSUE #142: Latest Swift syntax
When using `pod lib create NewLib` and opening the project in XCode, I am prompted to update to the latest Swift syntax.

screen shot 2015-12-20 at 10 15 51 pm

### PR DESCRIPTION
**Pull request by fulldecent**
fixes #142

### CODE REVIEW

File: templates/ios/Example/PROJECT.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -29,12 +29,12 @@
 			containerPortal = 6003F582195388D10070C39A /* Project object */;
 			proxyType = 1;
 			remoteGlobalIDString = 6003F589195388D20070C39A;
-			remoteInfo = "PROJECT";
+			remoteInfo = PROJECT;
```
Comment (orta) [inline templates/ios/Example/PROJECT.xcodeproj/project.pbxproj]: Thanks!

So a few of these changes will break when people use spaces in their pod names, which they do. Any chance you can add some quotes around these please? There's one or two down below too.

Comment (fulldecent) [inline templates/ios/Example/PROJECT.xcodeproj/project.pbxproj]: updated

File: templates/ios/Example/PROJECT.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -208,8 +208,8 @@
 			isa = PBXProject;
 			attributes = {
 				CLASSPREFIX = CPD;
-				LastUpgradeCheck = 0510;
-				ORGANIZATIONNAME = "PROJECT_OWNER";
+				LastUpgradeCheck = 0720;
+				ORGANIZATIONNAME = PROJECT_OWNER;
```
Comment (kylef) [inline templates/ios/Example/PROJECT.xcodeproj/project.pbxproj]: Remember this one too: `PROJECT_OWNER`.

Comment (orta) [inline templates/ios/Example/PROJECT.xcodeproj/project.pbxproj]: I sent a PR adding these :+1:

**Comment by fulldecent** [comment]
Thank you @orta this is now merged in here

File: templates/ios/Example/PROJECT.xcodeproj/project.pbxproj (hunk 2 above)
Comment (fulldecent) [inline templates/ios/Example/PROJECT.xcodeproj/project.pbxproj]: Thank you, merged

**Comment by orta** [comment]
Thanks @fulldecent - let's do this.

**Pull request merged**